src/main/java/edu/arizona/kra/proposaldevelopment/dao/ojb/PropDevRoutingStateDaoOjb.java

```diff
@@ -0,0 +1,473 @@
+            rs = ps.executeQuery();
+            if (rs.next()) { 
+                spsReviewerId = rs.getString(COL_SPS_REVIEWER_ID);
+            }
+            
+        } catch (SQLException sqle) {
+            LOG.error("SQLException: " + sqle.getMessage(), sqle);
+            throw sqle;
+        } catch (LookupException le) {
+            LOG.error("LookupException: " + le.getMessage(), le);
+            throw le;
+        } finally {
+            closeDatabaseObjects(rs, ps, conn, broker);
+        }
+        LOG.debug("getSPSReviewer Result="+spsReviewerId);
+        return spsReviewerId;
+    }
+
+
+    @Override
+    @Transactional
+    public void setSPSReviewer(String proposalNumber, String kcPersonId, String fullName) throws SQLException, LookupException {
+        LOG.debug("setSPSReviewer proposalNumber="+proposalNumber+" kcPersonId="+kcPersonId +" fullName="+fullName);
+        if (StringUtils.isEmpty(proposalNumber) || StringUtils.isEmpty(kcPersonId) ||  StringUtils.isEmpty(fullName)){
+            throw new IllegalArgumentException("PropDevRoutingStateDaoOjb: setSPSReviewer with null args!");
+        }
+        
+        String currentUserName = GlobalVariables.getUserSession().getPrincipalName();
+        Connection conn = null;
+        PreparedStatement ps1 = null, ps2 = null;
+        ResultSet rs = null;
+        String curIndQuery = CUR_IND_UPDATE_STMT.replace("$tablename", SPS_REV_TABLE_NAME);
+        PersistenceBroker broker = null;
+        try {
+            broker = this.getPersistenceBroker(true);
+            conn = broker.serviceConnectionManager().getConnection();
+            //first 'remove' last active version by setting cur_ind to 0
+            ps1 = conn.prepareStatement(curIndQuery);
+            ps1.setString(1, proposalNumber);
+            ps1.executeUpdate();
+            //insert new vale with cur_ind=1
+            ps2 = conn.prepareStatement(ADD_SPS_REVIEWER_QUERY);
+            ps2.setString(1, proposalNumber);
+            ps2.setString(2, kcPersonId);
+            ps2.setString(3, fullName);
+            ps2.setString(4, currentUserName);
+            ps2.executeUpdate();
+           
+        } catch (SQLException sqle) {
+            LOG.error("SQLException: " + sqle.getMessage(), sqle);
+            throw sqle;
+        } catch (LookupException le) {
+            LOG.error("LookupException: " + le.getMessage(), le);
+            throw le;
+        } finally {
+            closeDatabaseObjects(rs, ps1, conn, broker);
+            if ( ps2 != null ){
+                try {
+                    ps2.close();
+                } catch (SQLException ex) {
+                    LOG.warn("Failed to close PreparedStatement.", ex);
+                }
+            }
+        }
+        LOG.debug("setSPSReviewer:Finished.");
+        
+    }
+    
+    @Override 
+    public List<SPSReviewer> findSPSReviewers(Collection<String> principalIds) throws SQLException, LookupException {
```

Comment:
For now, SPSReviewer is just a simple bean to incapsulate a person name and id and send it back to the javascript in Propoosal Development Routing State Dashboard. I wanted to make this as fast as possible so I was trying to avoid any additional framework add-ons.
In the future, if we really need to use this class for additional purposes, we can definitely do that.